tests/channel_selection.rs

```diff
@@ -125,4 +125,63 @@ fn channel_selection() {
         .assert()
         .failure()
         .stderr("ERROR: Invalid Juliaup channel `1.8.6`. Please run `juliaup list` to get a list of valid channels and versions.\n");
+
+    // Now testing short channel matching
+    // At this point, installed channels are: 1.6.7, 1.7.3, 1.8.5
+
+    // Test that incomplete number matching does not autocomplete:
+    // https://github.com/JuliaLang/juliaup/pull/838#issuecomment-2206640506)
+    Command::cargo_bin("julia")
+        .unwrap()
+        .arg("+1.8")
+        .arg("-e")
+        .arg("print(VERSION)")
```

Comment:
These are also unnecessary

Reply:
This might be a stupid question but how does the test not get stuck in the REPL without the "-e"?

Reply:
Good point; that is necessary (or -v or something)

Reply:
I was asking because the tests pass without them and I was genuinely wondering what made it work. I'll add -v to be safe because though.